plugins/modules/source_control/gitlab/gitlab_project_members.py

```diff
@@ -93,6 +124,51 @@
     project: projectname
     gitlab_user: username
     state: absent
+
+- name: Add a list of Users to A GitLab project
+  gitlab_project_members:
```

Comment:
Same for the other ones.
```suggestion
  community.general.gitlab_project_members:
```